specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -852,6 +1013,323 @@
+          },
+      "criticalCount": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Count of updates available classified as critical"
+          },
+      "securityCount": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Count of updates available classified as security"
+          },
+      "otherCount": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Count of updates available that are classified as neither critical or security"
+          },
+      "approvalSource": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Source of approval"
+          },
+      "latestAssessmentTimeTaken": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Time taken for the latest assessment to complete on the computer"
+          },
+      "lastUpdateApplied": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Last update applied"
+          },
+      "updateAssessmentErrors": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Errors that occurred during assessment, if any"
+          },
+      "updateAssessmentErrorStackTrace": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Stack Trace for any terminal error during assessment, if any"
+          },
+      "lastRefreshDate": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Last refresh date"
+          }
+        }
+      },
+    "SoftwareUpdatesList": {
+        "properties": {
+          "value": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/SoftwareUpdate"
+            },
+            "description": "The collection of software updates found on the computer."
+          }
+        },
+        "description": "The collection of software updates found on the computer."
+      },
+      "SoftwareUpdate": {
+        "properties": {
+          "properties": {
+            "x-ms-client-flatten": true,
+            "$ref": "#/definitions/SoftwareUpdateProperties",
+            "description": "Properties of the software update."
+          }
+        },
+        "allOf": [
+          {
+          "$ref": "../../common/v1/definitions.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Describes a specific software update on the given virtual machine."
+      },
+      "SoftwareUpdateProperties": {
+        "description": "Properties of the software update object.",
+        "properties": {
+          "updateState": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Needed / Installed"
+          },
+          "availableOnComputerSince": {
```

Comment:
i think this should be ISO 8601 compliant, please add format property with value as date-time.

Reply:
Done